jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/complex/menu/BasicMenuTests.java

```diff
@@ -0,0 +1,59 @@
+package io.github.epam.angular.tests.elements.complex.menu;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static com.epam.jdi.light.angular.elements.common.NestedDropdownMenu.BASIC_MENU_VALUES;
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.sections.MenuSection.basicMenu;
+import static io.github.com.pages.sections.MenuSection.basicMenuSelectedOption;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+
+public class BasicMenuTests extends TestsInit {
+
+    @BeforeTest
+    public void before() {
+        shouldBeLoggedIn();
+        angularPage.shouldBeOpened();
+        basicMenu.show();
+    }
+
+    @Test
+    public void basicMenuTest() {
+        basicMenu.is().displayed();
+    }
+
+    @Test
+    public void checkBasicMenuIsExpandedTest() {
```

Comment:
This test will only pass if on a single run. In case you run another test right after that, it will fail. The reason is that you are not resetting the SUT to its original state. That is, if the menu remains expanded, you cannot execute the expand() method again after that.

Reply:
Fixed, thanks.